picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
+        longdesc=N_('The MusicBrainz Identifier (MBID) for the release group.'),
     ),
     TagVar(
         'musicbrainz_trackid',
         shortdesc=N_('MusicBrainz Track Id'),
+        longdesc=N_('The MusicBrainz Identifier (MBID) for the track.'),
     ),
     TagVar(
         'musicbrainz_tracknumber',
         shortdesc=N_('FIXME:musicbrainz_tracknumber'),
+        longdesc=N_(
+            'The track number written as on the MusicBrainz release, such as vinyl numbering (A1, A2, etc.)'
+        ),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'musicbrainz_workid',
         shortdesc=N_('MusicBrainz Work Id'),
+        longdesc=N_('The MusicBrainz Identifier (MBID) for the Work if a related work exists.'),
     ),
     TagVar(
         'musicip_fingerprint',
         shortdesc=N_('MusicIP Fingerprint'),
+        longdesc=N_('The MusicIP Fingerprint for the track.'),
+        not_from_mb=True,
     ),
     TagVar(
         'musicip_puid',
         shortdesc=N_('MusicIP PUID'),
+        longdesc=N_('The MusicIP PUIDs associated with the track.'),
+        not_from_mb=True,
     ),
     TagVar(
         'originalalbum',
         shortdesc=N_('Original Album'),
+        longdesc=N_(
+            'The release title of the earliest release in the release group intended for the title of the '
+            'original recording.'
+        ),
+        not_from_mb=True,
     ),
     TagVar(
         'originalartist',
         shortdesc=N_('Original Artist'),
+        longdesc=N_(
+            'The track artist of the earliest release in the release group intended for the performers of '
+            'the original recording.'
+        ),
+        not_from_mb=True,
     ),
     TagVar(
         'originaldate',
         shortdesc=N_('Original Release Date'),
+        longdesc=N_(
+            'The original release date in the format YYYY-MM-DD. By default this is set to the earliest '
+            'release in the release group. This can provide, for example, the release date of the vinyl '
+            'version of what you have on CD.\n\nThis is the same information provided in the '
+            '`%_releasegroup_firstreleasedate%` variable, and is consistent across all tracks in the '
+            'release. If you prefer to have this tag populated with the date of the earliest recording of '
+            'the track in the database, which will likely be different for each track in the release, '
+            'this can be achieved by enabling a one-line tagging script as:\n\n'
+            '`$set(originaldate,%_recording_firstreleasedate%)`\n\nBe aware that setting this can cause a '
+            'release to be scattered across multiple directories if you use `%originaldate%` as part of '
+            'the path portion of your file naming script.\n\n**Note:** If you are storing tags in MP3 files as '
+            'ID3v2.3 then the original date can only be stored as a year.'
```

Comment:
I haven't seen this in action yet, but it seems to me this might be a bit too verbose for a tooltip, with a lot of very specific technical details.

Reply:
Yes, that's far too long for a tool tip, if we really need a long documentation we could add an attribute to store it (`extralongdesc` or whatever) but it shouldn't be displayed in the tooltip (it could in an internally generated documentation page for example).